### PR TITLE
Use requests session per request handler instance

### PR DESCRIPTION
**Pull request by abn**
Resolves: Issue #35

### CODE REVIEW

File: ubersmith/api.py [hunk 1]
```diff
@@ -3,7 +3,7 @@
 import time
 from ubersmith.compat import total_ordering, file_type
 
-import requests
+from requests import session
```
Comment (theY4Kman) [inline ubersmith/api.py]: Would you keep this as `import requests`? I'd find `self._session = requests.session()` below much more readable.

Comment (abn) [inline ubersmith/api.py]: Done

**Comment by theY4Kman** [comment]
I apologize for the long delay on this. I am on vacation now, but when I return, I will try to evaluate and merge this. I no longer have access to an Ubersmith instance, so I will rely on my better judgment to vet it.

**Comment by abn** [comment]
@theY4Kman any updates on this one?

**Comment by theY4Kman** [comment]
@abn I merged this today, with one slight adjustment: you can pass a `requests.Session()` to `RequestHandler`, so you can preconfigure your session, instead of modifying it after the `RequestHandler` has been instantiated. See commit 67db0901fd16814f70eaafb9c82ee9ffb7c41680.

It's available in PyPI as version 0.4.0. Better late than never ;)

**Comment by abn** [comment]
@theY4Kman awesome thanks for that. Change looks good.

**Pull request closed without merging**